src/gamepad/linux/linux-guid-helpers.vala

```diff
@@ -0,0 +1,31 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class LibGamepad.LinuxGuidHelpers : Object {
+	public static string from_dev (Libevdev.Evdev device) {
```

Comment:
Earlier this was also being used in raw-gamepad-monitor, so it is here. but moved